icu4j/main/classes/core/src/com/ibm/icu/impl/LocaleFallbackTables.java

```diff
@@ -0,0 +1,562 @@
+        { "sga", "Ogam" },
+        { "sgw", "Ethi" },
+        { "shi", "Tfng" },
+        { "shn", "Mymr" },
+        { "shu", "Arab" },
+        { "si", "Sinh" },
+        { "skr", "Arab" },
+        { "smp", "Samr" },
+        { "sog", "Sogd" },
+        { "sou", "Thai" },
+        { "sr", "Cyrl" },
+        { "srb", "Sora" },
+        { "srx", "Deva" },
+        { "swb", "Arab" },
+        { "swv", "Deva" },
+        { "syl", "Beng" },
+        { "syr", "Syrc" },
+        { "ta", "Taml" },
+        { "taj", "Deva" },
+        { "tcy", "Knda" },
+        { "tdd", "Tale" },
+        { "tdg", "Deva" },
+        { "tdh", "Deva" },
+        { "te", "Telu" },
+        { "tg", "Cyrl" },
+        { "tg_PK", "Arab" },
+        { "th", "Thai" },
+        { "thl", "Deva" },
+        { "thq", "Deva" },
+        { "thr", "Deva" },
+        { "ti", "Ethi" },
+        { "tig", "Ethi" },
+        { "tkt", "Deva" },
+        { "trw", "Arab" },
+        { "tsd", "Grek" },
+        { "tsf", "Deva" },
+        { "tsj", "Tibt" },
+        { "tt", "Cyrl" },
+        { "tts", "Thai" },
+        { "txg", "Tang" },
+        { "txo", "Toto" },
+        { "tyv", "Cyrl" },
+        { "udi", "Aghb" },
+        { "udm", "Cyrl" },
+        { "ug", "Arab" },
+        { "ug_KZ", "Cyrl" },
+        { "ug_MN", "Cyrl" },
+        { "uga", "Ugar" },
+        { "uk", "Cyrl" },
+        { "unr", "Beng" },
+        { "unr_NP", "Deva" },
+        { "unx", "Beng" },
+        { "ur", "Arab" },
+        { "uz_AF", "Arab" },
+        { "uz_CN", "Cyrl" },
+        { "vai", "Vaii" },
+        { "wal", "Ethi" },
+        { "wbq", "Telu" },
+        { "wbr", "Deva" },
+        { "wni", "Arab" },
+        { "wsg", "Gong" },
+        { "wtm", "Deva" },
+        { "wuu", "Hans" },
+        { "xco", "Chrs" },
+        { "xcr", "Cari" },
+        { "xlc", "Lyci" },
+        { "xld", "Lydi" },
+        { "xmf", "Geor" },
+        { "xmn", "Mani" },
+        { "xmr", "Merc" },
+        { "xna", "Narb" },
+        { "xnr", "Deva" },
+        { "xpr", "Prti" },
+        { "xsa", "Sarb" },
+        { "xsr", "Deva" },
+        { "yi", "Hebr" },
+        { "yue", "Hant" },
+        { "yue_CN", "Hans" },
+        { "zdj", "Arab" },
+        { "zgh", "Tfng" },
+        { "zh", "Hans" },
+        { "zh_AU", "Hant" },
+        { "zh_BN", "Hant" },
+        { "zh_GB", "Hant" },
+        { "zh_GF", "Hant" },
+        { "zh_HK", "Hant" },
+        { "zh_ID", "Hant" },
+        { "zh_MO", "Hant" },
+        { "zh_PA", "Hant" },
+        { "zh_PF", "Hant" },
+        { "zh_PH", "Hant" },
+        { "zh_SR", "Hant" },
+        { "zh_TH", "Hant" },
+        { "zh_TW", "Hant" },
+        { "zh_US", "Hant" },
+        { "zh_VN", "Hant" },
+        { "zhx", "Nshu" },
+        { "zkt", "Kits" },
+    };
+
+    //======================================================================
+    // Parent locale table
+    public static final String[][] PARENT_LOCALE_TABLE = {
+        { "az_Arab", "root" },
```

Comment:
Shouldn't we use "" instead of "root" ?

Reply:
I like "root."  It matches what's in the original resource files, and I think it makes it clearer what we're doing.